app/api-v1/routes/recipe/{id}/creation.js

```diff
@@ -70,7 +70,7 @@ module.exports = function () {
         name: 'id',
         allowEmptyValue: false,
         schema: {
-          $ref: '#/components/schemas/ObjectReference',
+          $ref: '#/components/schemas/ChainAction',
```

Comment:
this looks wrong. I really think this should be an ObjectReference. A `ChainAction` is an object